application/config/config_test.go

```diff
@@ -300,3 +300,39 @@ func Test_IsTelemetryEnabled(t *testing.T) {
 	assert.False(t, c.IsTelemetryEnabled())
 	assert.True(t, c.Engine().GetConfiguration().GetBool(configuration.ANALYTICS_DISABLED))
 }
+
+func TestSnykUiEndpoint(t *testing.T) {
+	c := New()
+	t.Run("Default Api Endpoint with /api prefix", func(t *testing.T) {
+		uiEndpoint := c.SnykUi()
+		assert.Equal(t, "https://snyk.io", uiEndpoint)
```

Comment:
@cat2608 I'm not really sure here if this is right though. 
If the the API endpoint doesn't have the `api` or `app` prefix. Is it correct to to use `snyk.io` as the SnykUi endpoint?

Maybe we should always append `app` as a prefix subdomain ?